### PR TITLE
Complementary python script inline in the cwl workflow

### PR DESCRIPTION
**Pull request by alexiswl**
Python script to match: https://github.com/umccr-illumina/cwl-iap/pull/130

### CODE REVIEW

File: scripts/showcase/samplesheet-by-override-cycles.py [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+    # Arguments
+    parser.add_argument("--samplesheet-csv", required=True,
+                        help="Path to samplesheet csv")
+    parser.add_argument("--out-dir", required=False,
+                        help="Output directory for samplesheets, set to cwd if not specified")
+    parser.add_argument("--samples", required=False,
+                        help="Comma separated values to match override-cycles arg")
+    parser.add_argument("--override-cycles", required=False,
+                        help="Comma separated values to match samples arg")
```
Comment (skanwal) [inline scripts/showcase/samplesheet-by-override-cycles.py:55]: To confirm, if these arguments are "None", the same input samplesheet is returned?

**Review by skanwal — APPROVED** [review]
Thanks. One quick question for my context otherwise works great.

File: scripts/showcase/samplesheet-by-override-cycles.py (hunk 1 above)
Comment (alexiswl) [reply scripts/showcase/samplesheet-by-override-cycles.py:55]: Yes, unless `--v2` is also specified, in which case the samplesheet headers are converted for `Settings` and `Data`

Comment (alexiswl) [reply scripts/showcase/samplesheet-by-override-cycles.py:55]: And that same samplesheet is placed in the directory specified by `--out-dir` as `SampleSheet.csv`

**Pull request merged**